packages/app-next/src/App.tsx

```diff
@@ -91,16 +91,18 @@ const homePageExtension = createExtension({
 });
 
 const signInPage = createSignInPageExtension({
-  id: 'signInPage',
+  name: 'guest',
   loader: async () => (props: SignInPageProps) =>
     <SignInPage {...props} providers={['guest']} />,
 });
 
 const scmAuthExtension = createApiExtension({
+  name: 'scm-auth',
   factory: ScmAuth.createDefaultApiFactory(),
 });
 
 const scmIntegrationApi = createApiExtension({
+  name: 'scm-integration',
```

Comment:
Oops, I force-pushed away the changes from #21545 by mistake, brought them back in so that this isn't an issue in the particular spot. Agreed on the naming point though, let's continue the naming discussion in the next couple of days as this does not completely close that.